### PR TITLE
Update xml-instructions.md to remove unnecessary html tag

### PR DESCRIPTION
**Pull request by hgajjar**
## This PR is a:

- [ ] New topic
- [ ] Content update
- [X] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary
Removed extra HTML tag which is being displayed in browser, and fixed a misleading xml tag for an xml attribute.

## Additional information

List all affected URLs 

https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/layouts/xml-instructions.html
https://devdocs.magento.com/guides/v2.2/frontend-dev-guide/layouts/xml-instructions.html
https://devdocs.magento.com/guides/v2.1/frontend-dev-guide/layouts/xml-instructions.html

whatsnew
Corrected the syntax for the `remove` attribute used with the `<referenceBlock>`  and `<referenceContainer`  [layout instructions](https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/layouts/xml-instructions.html).

### CODE REVIEW

**Comment by magento-cicd2** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md [hunk 1]
```diff
@@ -206,7 +206,7 @@ Sets the declared block or container element as a child of another element in th
 ### remove {#fedg_layout_xml-instruc_ex_rmv}
 
 `<remove>` is used only to remove the static resources linked in a page `<head>` section.
-For removing blocks or containers, use the `<remove>` attribute for [`<referenceBlock>` and `<referenceContainer>`](#fedg_layout_xml-instruc_ex_ref).
+For removing blocks or containers, use the `remove` attribute for [`<referenceBlock>` and `<referenceContainer>`](#fedg_layout_xml-instruc_ex_ref).
```
Comment (meker12) [inline guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md]: @hgajjar  The incorrect syntax for `<remove>` is also in the v2.1 file,  (guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md).  Can you add the correction to that file as well?

File: guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md [hunk 2]
```diff
@@ -126,7 +126,7 @@ The `<action>` instruction is deprecated. If the method implementation allows, u
 
 Calls public methods on the block API.
 
-**Details:** Used to set up the execution of a certain method of the block during block generation; the `<action>` node must be located in the scope of the `<block>` node.</p>
+**Details:** Used to set up the execution of a certain method of the block during block generation; the `<action>` node must be located in the scope of the `<block>` node.
```
Comment (meker12) [inline guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md]: v2.1 of the file still has the <p> and </p> html tags. Can you remove those tags as well?

**Comment by hgajjar** [comment]
@meker12 Code review changes are done, kindly review, thanks.

**Comment by meker12** [comment]
running tests

**Pull request merged**